pkg/broker/broker.go

```diff
@@ -185,14 +186,37 @@ func (a AnsibleBroker) Bootstrap() (*BootstrapResponse, error) {
 		a.log.Error("Something went real bad trying to delete batch specs... - %v", err)
 		return nil, err
 	}
+	specs = []*apb.Spec{}
 
-	if specs, imageCount, err = a.registry.LoadSpecs(); err != nil {
-		return nil, err
+	//Load Specs for each registry
+	registryErrors := []error{}
+	for _, r := range a.registry {
```

Comment:
I agree it could. I think the idea is we can go back and do this, while the way it is implemented now is easier and more readable for the first pass at the feature.